setup.py

```diff
@@ -210,7 +207,28 @@ def find_sources(self):
     "long_description_content_type": "text/markdown",
     "include_package_data": True,
     "zip_safe": False,
-    "install_requires": required,
+    "install_requires": [
+        "aiofiles>=0.8.0",
+        "aiohttp>=3.8.1",
+        "alembic>=1.8.0",
+        "click",
+        "cloudpickle>=2.0.0",
+        "dask[distributed]>=2022.6.0",
+        "fastapi>=0.93.0",
+        "furl",
+        "networkx",
+        "psutil>=5.9.0",
+        "pydantic>=1.10.1",
+        "python-socketio",
+        "requests>=2.24.0",
+        "simplejson",
+        "sqlalchemy>=1.4.37",
+        "sqlalchemy_utils",
+        "toml",
+        "uvicorn[standard]==0.18.3",
+        "watchdog",
+        "werkzeug>=2.0.3",
+    ],
```

Comment:
Let's put this entire list in a variable instead somewhere early in the file. Will make adding/updating things easier.